docs/hosting-a-server-with-northstar/dedicated-server/hosting-on-windows.md

```diff
@@ -0,0 +1,47 @@
+# Hosting on Windows
+
+## Hardware
+
+{% hint style="info" %}
+TODO: If you have experience in hosting Northstar servers and want to share your knowledge, please open a [pull request](https://github.com/R2Northstar/NorthstarWiki/pulls).
+{% endhint %}
+
+## Minimum requirements
+
+The current minimum requirements are as follows:
+
+**Requirements per server:**
+
+- 8GB free disk space (5GB of the game and 3 for Origin/EA app)
+- 3+ cores (2 might work, though)
+- 3GB+ total memory (RAM or swap will do)
+- Windows 8.1 or better (and built-in WARP [use `-softwared3d11`] or a graphics card)
+
+**Per instance:**
+
+- 2GB RAM (this is what's actually used; it does drop to ~1.2GB after a bit)
+- 15 Mbps network upload, but 10 is workable, 25 if you want to avoid players getting disconnected when going back to the lobby after a match
+
+**Note:** It is recommended to surpass the listed requirements. Currently the number of available servers covers the daily playerbase more than enough. If you're planning to host public servers for the community it is therefore recommended to either fill a niche (like a gamemode that is so popular that all existing servers are full) or provide a better service than existing hosts (less lag, more stable, etc.).
+
+
+## Setup
+
+Dedicated servers allow you to host a Northstar server without having to use a full client, making them more lightweight and easier to host for longer periods of time. Dedicated servers are still in development for Northstar, so while they do work, expect some bugs and general jank.\
+To start a dedicated server on Northstar, launch NorthstarLauncher.exe with the argument `-dedicated`, this can be done manually, but releases also provide the batch file `r2ds.bat`, which will also do this.\
+When using a dedicated server, arguments are read from `ns_startup_args_dedi.txt`, rather than `ns_startup_args.txt`.
+
```

Comment:
```suggestion

You can also follow a video guide for this process from [this YouTube link](https://www.youtube.com/watch?v=EZ3w2Nl9SZo)

```
if doing this, also delete the entire `guides` folder from `docs/hosting-a-server-with-northstar/`, and remove line 46 from [`SUMMARY.md`](https://github.com/NoCatt/NorthstarWiki/blob/re-serverhosting/docs/SUMMARY.md?plain=1#L46)